spec/test-outputs/www-integration.out.vcl

```diff
@@ -159,6 +159,39 @@ sub vcl_recv {
     set req.http.GOVUK-Account-Session = req.http.Cookie:__Host-govuk_account_session;
   }
 
+  if (req.http.Cookie ~ "cookies_policy") {
+    if (req.http.Cookie:cookies_policy ~ "%22usage%22:true" && req.http.User-Agent !~ "^GOV\.UK Crawler Worker" ) {
+      if (table.lookup(active_ab_tests, "StartABusinessSegment") == "true") {
+        declare local var.sab_variant STRING;
+
+        # Always send this header if they've already got a cookie
+        # This way, the cookie expiry gets updated in DELIVER, even if it's not a SaB page.
+        # Thus sessions that that have >=2 SaB pages but more than half an hour apart
+        # will be included.
+        if (req.http.Cookie ~ "ABTest-start_a_business_segment=(A|B|C)") {
+          set var.sab_variant = re.group.1;
+          set req.http.GOVUK-ABTest-StartABusinessSegment = var.sab_variant;
+        } else {
+          # Cookie not set, default to A
+          set var.sab_variant = "A";
+        }
+
+        if (table.lookup(start_a_business_ab_test_base_paths, querystring.remove(req.url)) == "true") {
+          # Header so rendering app will know it's a SaB page (and the config is in just one place)
+          set req.http.GOVUK-ABTest-IsStartABusinessPage = "true";
+          # Segment header
+          set req.http.GOVUK-ABTest-StartABusinessSegment = var.sab_variant;
+        }
+
+        if (querystring.remove(req.url) ~ "next-steps-for-your-business" || std.strlen(querystring.get(req.url, "hide-intervention")) > 0) {
```

Comment:
Worth double checking that you may have both `GOVUK-ABTest-IsStartABusinessPage = "true"` set as well as going into this final `if` block - this is desired right? (assume it is)

Reply:
Yes, this is desired, good question though